Dockerfile

```diff
@@ -1,6 +0,0 @@
-
-FROM nginx:alpine
-MAINTAINER cryptic-game
```

Comment:
Is there a reason for this deletion?

Reply:
Thought it's unneccessary, because it belongs to the old side

Reply:
No this is necessary for deployment

Reply:
Then you have to add it to the new app, I don't know how this works

Reply:
Okay, we will just ignore this for a moment :+1: